src/Waives.Http/Document.cs

```diff
@@ -135,6 +135,25 @@ public async Task<ClassificationResult> Classify(string classifierName)
         /// <param name="extractorName">The name of the extractor to use.</param>
         /// <returns>The results of the extraction operation.</returns>
         public async Task<ExtractionResults> Extract(string extractorName)
+        {
+            var extractionResponse = await DoExtraction(extractorName).ConfigureAwait(false);
+            return await extractionResponse.ReadAsAsync<ExtractionResults>().ConfigureAwait(false);
+        }
+
+        public async Task<Stream> Redact(string extractorName)
+        {
+            var extractionResponse = await DoExtraction(extractorName).ConfigureAwait(false);
```

Comment:
✋ I'm confused about how this combination of extraction and redaction requests could be working. The extraction request should have an `Accept` header with the value `application/vnd.waives.requestformats.redact+json` in order to get a response in the correct format. But I don't see that header being set anywhere. Am I missing something? Are you seeing this actually working?

Reply:
Oh, good catch. Missed that in the docs, etc. I'll do a proper test 😉